orc8r/cloud/go/blobstore/storage.go

```diff
@@ -217,30 +216,20 @@ type LoadCriteria struct {
 	LoadValue bool
 }
 
-// GetTKsFromBlobs converts blobs to their associated type and key.
-func GetTKsFromBlobs(blobs []Blob) []storage.TypeAndKey {
-	tks := make([]storage.TypeAndKey, 0, len(blobs))
-	for _, blob := range blobs {
+// TKs converts blobs to their associated type and key.
+func (bs Blobs) TKs() []storage.TypeAndKey {
```

Comment:
Move these `Blobs` methods directly under the `Blobs` type declaration, where Blobs.GetKeys is located